collector/cpu_linux.go

```diff
@@ -57,6 +58,11 @@ func NewCPUCollector() (Collector, error) {
 			"Seconds the cpus spent in each mode.",
 			[]string{"cpu", "mode"}, nil,
 		),
+		cpuGuest: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, cpuCollectorSubsystem, "guest_seconds_total"),
+			"Seconds the cpus spent for guests (VMs) in each mode.",
```

Comment:
The above metric uses "in" rather than "for"

Reply:
How about: `Seconds the cpus spent in guests (VMs) for each mode.`

Reply:
Sure.